addons/web/static/src/js/views/list/list_editable_renderer.js

```diff
@@ -1335,7 +1335,9 @@ ListRenderer.include({
      * Set a maximum width on the largest columns in the list in case the table
      * is overflowing. The idea is to shrink largest columns first, but to
      * ensure that they are still the largest at the end (maybe in equal measure
-     * with other columns).
+     * with other columns). Button columns aren't impacted by this function, as
+     * we assume that they can't be squeezed (we want all buttons to always be
+     * available, not being replaced by ...).
```

Comment:
At first I though the sentence wasn't finished. Perhaps add quotes or replace with "ellipsis" ?